config-connector/solutions/iam/kpt/service-account/README.md

```diff
@@ -0,0 +1,52 @@
+Service Account
+==================================================
+
+# NAME
+
+  service-account
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create a service account in your desired project, and grant a specific member a role (default to iam.serviceAccountKeyAdmin) for accessing the service account that just created.
+
```

Comment:
You have missed the "roles/". It defaults to `roles/iam.serviceAccountTokenCreator`.

Reply:
Thanks for the catch. Fixed.